schemas/context/experienceevent.schema.json

```diff
@@ -118,6 +118,12 @@
           "title": "Advertising",
           "$ref": "https://ns.adobe.com/xdm/context/advertising",
           "description": "The information related to advertising activity related to the experience event"
+        },
+        "xdm:media": {
```

Comment:
The thought process on this was that this will describe all media-related data from an experience event - static, timed & interactive media. 

Atm it only contains timed media, as this is the only type of media tracked through Media Analytics. In the future it should be extended with other media types when products that use such types of media integrate their schemas.

Reply:
in that case - media is the right term!